### ISSUE TITLE
[dev.icinga.com #6167] Dropdown tabs are not accessible for keyboard navigation

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#644]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/6167

**Created by tgelf on 2014-05-09 09:14:04 +00:00**

Assignee: _tgelf_
Status: _Resolved_  (closed on _2014-06-20 19:22:53 +00:00_)
Target Version: _(none)_
Last Update: _2014-06-20 19:22:53 +00:00 (in Redmine)_

---

There is no other way then "mouse-hovering" to show the dropdown menu. Adding "click support" should fix this.

Cheers,
Thomas

**Changesets**

_2014-05-09 11:49:15 +00:00_ by _tgelf_ 19220a5a44c412721c3aa17602e2722457173286

    Initial "click" handling for dropdown tabs
    
    Still uncomplete, there are a number of edge cases that need to be
    handled. Keyboard navigation is one of them, hover/click distinction
    is another one.
    
    refs #6167
